uPortal-web/build.gradle

```diff
@@ -12,6 +12,7 @@ dependencies {
     compile project(':uPortal-rendering')
 
     compile "com.google.visualization:visualization-datasource:${googleVisualizationVersion}"
+    compile "joda-time:joda-time:${jodaTimeVersion}"
```

Comment:
I'm missing something here...
Why is this dependency needed now? It doesn't appear to be used in this PR.

Reply:
My IDE was showing errors about this being a missing dependency. It is a transitive dependency, but it should be included as it is being used by code in this module.

Reply:
:confused: Doesn't Gradle include all transitive dependencies unless they are explicitly excluded?
see https://docs.gradle.org/current/userguide/dependency_constraints.html#sec:direct-vs-transitive-deps and https://docs.gradle.org/current/userguide/dependency_downgrade_and_exclude.html

Reply:
If code relies on a dependency the module should include that rather than relying on a transitive dependency of another dependency to bring that in. Right?

Reply:
Ah, that was the missing puzzle piece, it is directly used.
https://github.com/Jasig/uPortal/blob/647b5e072aab35cacdcaa6aab0bc2c0dd429a613/uPortal-web/src/main/java/org/apereo/portal/portlet/dao/jpa/PortalCookieImpl.java#L42